config/locales/en.yml

```
@@ -113,7 +113,7 @@ en:
             label: Individual
         custom_select_error: Select the best way to describe your business
       product_details:
-        title: Tell us about each product you’re offering
+        title: Tell us about the product you're offering
```

```suggestion
        title: Tell us about the product you’re offering
```

I think we need to use the special `’` for these, no?
else they might clash with string closures?

right @issyl0 ?